src/io/iter_sparse_prefetcher.h

```diff
@@ -126,15 +126,15 @@ class SparsePrefetcherIter : public PrefetcherIter {
   virtual bool Next(void) {
     return PrefetcherIter::Next();
   }
-  virtual const DataBatch &Value(void) const {
+  virtual const DataBatch& Value(void) const {
     return PrefetcherIter::Value();
   }
 
-  virtual const NDArrayStorageType GetStorageType(bool is_data) const {
+  virtual NDArrayStorageType GetStorageType(bool is_data) const {
```

Comment:
Why remove const?

Reply:
I replied below. It makes very little sense.